tests/integration.rs

```diff
@@ -3091,6 +3091,42 @@ mod parallel {
         handle_child_output(r, &output);
     }
 
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    fn test_dmi_uuid() {
+        let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+        let guest = Guest::new(Box::new(focal));
+
+        let mut child = GuestCommand::new(&guest)
+            .args(&["--cpus", "boot=1"])
+            .args(&["--memory", "size=512M"])
+            .args(&["--kernel", direct_kernel_boot_path().to_str().unwrap()])
+            .args(&["--cmdline", DIRECT_KERNEL_BOOT_CMDLINE])
+            .args(&["--platform", "uuid=1e8aa28a-435d-4027-87f4-40dceff1fa0a"])
```

Comment:
would you take a look at also adding this option to the unit tests?

Reply:
The SMBIOS unit tests are non-existant. I like having this integration test as it also checks the byte swapping is working correctly by the guest seeing the same UUID.